entries/event.which.xml

```diff
@@ -13,7 +13,10 @@
     <desc>Log which key was depressed.</desc>
     <code><![CDATA[
 $( "#whichkey" ).on( "keydown", function( event ) {
-  $( "#log" ).html( event.type + ": " +  event.which );
+    $( "#log" ).append( event.type + ": " + event.which + "<br>" );
+});
+$( "#whichkey" ).on( "keypress", function( event ) {
+    $( "#log" ).append( event.type + ": " + event.which + "<br>" );
```

Comment:
I am confused as to what you are asking for...which why I gave you write access to the repository...